src/core/server/logging/README.md

```diff
@@ -324,12 +327,33 @@ Here is what we get with the config above:
 | server           | console, file            | fatal |
 | optimize         | console                  | error |
 | telemetry        | json-file-appender       | all   |
+| metrics.ops      | console                  | debug |
+
 
+### Dedicated loggers
 
 The `root` logger has a dedicated configuration node since this context is special and should always exist. By 
 default `root` is configured with `info` level and `default` appender that is also always available. This is the 
 configuration that all custom loggers will use unless they're re-configured explicitly.
 
+The `metrics.ops` logger is configured with `debug` level and will automatically output sample system and process information at a regular interval.
+The metrics that are logged are a subset of the data collected and are formatted in the log message as follows:
+
+| Ops formatted log property | Location in metrics service | Log units
+| :------------------------- | :-------------------------- | :-------------------------- |
+| memory | process.memory.heap.used_in_bytes | MB |
```

Comment:
The `units` column is a bit confusing here since `process.memory.heap.used_in_bytes` is of course a number in bytes, however the log message could be formatted in other units since the string from `numeral.format` [varies based on the value](http://numeraljs.com/#format).

So technically the message could be something other than `MB`. (Although in practice is most likely to be `MB` or `GB` for most people).

I'm not sure what the clearest way to explain this is, though. WDYT?

Reply:
I'll change that to say it's variable and link to the docs you pointed out.